x/virtualgroup/keeper/keeper.go

```diff
@@ -167,6 +167,7 @@ func (k Keeper) DeleteGVG(ctx sdk.Context, primarySp *sptypes.StorageProvider, g
 		k.paymentKeeper.IsEmptyNetFlow(ctx, sdk.MustAccAddressFromHex(gvgFamily.VirtualPaymentAddress)) &&
 		!ctx.IsUpgraded(upgradetypes.Manchurian) {
 		store.Delete(types.GetGVGFamilyKey(gvg.FamilyId))
+		k.DeleteGVGFamilyWithinSP(ctx, primarySp.Id, gvg.FamilyId)
```

Comment:
not needed

Reply:
The above GetGVGFamilyKey deletes this VGF. Logically, the list of VGFs under this SP should also have the corresponding VGF removed. Why is this not necessary?

Reply:
deleted